caluma/caluma_form/filters.py

```diff
@@ -486,7 +482,7 @@ class VisibleAnswerFilter(Filter):
     field_class = BooleanField
 
     def filter(self, qs, value):
-        if not value or not qs.exists():
+        if value in EMPTY_VALUES or not qs.exists():  # pragma: no cover
```

Comment:
I was just gonna suggest removing the `qs.exists()` check. But given that it triggers loading a whole document structure and evaluating the visibility JEXL, maybe we should add a comment instead to explain why we're doing this here, while we're at it :-) Might be a bit unrelated but useful for the next person to look at this

Reply:
Added a comment